manifests/README.md

```diff
@@ -0,0 +1,40 @@
+# TL;DR
+If you want to skip any customization, you can deploy Kubeflow Pipelines by running
+```
+export PIPELINE_VERSION=0.1.20
+kubectl create -f https://raw.githubusercontent.com/kubeflow/pipelines/$PIPELINE_VERSION/manifests/namespaced-install.yaml
+```
+
+You might lack the permission to create role and command might partially fail. If so, bind your account as cluster admin.
+(Or role creator in your namespace)
+```
+kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
+```
+
+# Customization
+
+## Change deploy namespace
```

Comment:
Let's keep the base unchanged. It is recommend to create overlays and add customization in the overlay.

Reply:
OK I'd picture the base/ dir to be unchanged, and people add overlay if needed. regarding this comment, do you just want me to call this out, or something else i need to do?

Reply:
Call this out and explain that with all customization in the overlay, it is easy to understand what has been changed.